src/components/header/modals/account-unification/Help.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="wrapper--help">
+    <div class="text--title">Need help?</div>
+    <dl class="list--help">
+      <dt>[place holder] I don’t know which address I need to use?</dt>
+      <dd>
+        You don’t need to think of which one, balance of the both accounts are the same. You don’t
```

Comment:
@ayumitk 
Let's use i18

Reply:
@impelcrypto I moved all texts to the en-US lang file and use i18n ([4ed7eb1](https://github.com/AstarNetwork/astar-apps/pull/935/commits/4ed7eb1a34ac5511f8b59da16914fb93f7d1e958), [2b6db2a](https://github.com/AstarNetwork/astar-apps/pull/935/commits/2b6db2a39566f3546912a141645f9dcb53f38f83))